packages/unchained-client/src/evm/parser/erc20.ts

```diff
@@ -34,6 +34,16 @@ export class Parser<T extends Tx> implements SubParser<T> {
 
     const decoded = this.abiInterface.parseTransaction({ data: tx.inputData })
 
+    let approveValue: string | null = null
+
+    switch (txSigHash) {
+      case this.supportedFunctions.approveSigHash:
+        approveValue = decoded?.args.amount?.toString()
```

Comment:
Note: ERC20 parsing works for ZRX, CowSwap, FOXy staking and UniV2 LPs. From the testing I've done, only Yearn requires separate parsing.

Reply:
IIFE to get rid of let and ternary below

Reply:
Done in 338d2e7a9e8a1d522906358acb0913dae03ac334 / f91cd8c179d2b2ef0ca004c96bc16b9f18332464